mesonbuild/modules/gnome.py

```diff
@@ -875,7 +875,7 @@ def _unpack_args(arg, kwarg_name, kwargs, expend_file_state=None):
 
         return []
 
-    @permittedKwargs({'interface_prefix', 'namespace', 'object_manager', 'build_by_default',
+    @permittedKwargs({'interface_prefix', 'namespace', 'extra_args', 'object_manager', 'autocleanup', 'build_by_default',
```

Comment:
Left-overs from autocleanup commit.

Reply:
This branch has both commits - I can separate them into two MPs with the second one needing to be rebased or this can be merged after the autocleanup MP.